centrifuge-app/src/pages/Pool/Assets/index.tsx

```diff
@@ -44,8 +44,8 @@ export function PoolDetailAssets() {
   if (!poolId) throw new Error('Pool not found')
 
   const pool = usePool(poolId)
-  const { data: loans, isLoading } = useLoans(poolId)
-  const { isLoading: isLoadingSnapshots, data: snapshots } = useAllPoolAssetSnapshots(poolId, new Date().toString())
+  const { data: loans } = useLoans(poolId)
+  const { data: snapshots } = useAllPoolAssetSnapshots(poolId, new Date().toString())
```

Comment:
```suggestion
  const { data: snapshots } = useAllPoolAssetSnapshots(poolId, new Date().toISOString().slice(0, 10))
```

Now this will fetch every render because the date will be different every time